services/web/server/src/simcore_service_webserver/socketio/handlers.py

```diff
@@ -153,6 +152,9 @@ async def disconnect(sid: str, app: web.Application) -> None:
             with managed_resource(user_id, client_session_id, app) as rt:
                 log.debug("client %s disconnected from room %s", user_id, sid)
                 await rt.remove_socket_id()
+            # signal same user other clients if available
+            await emit("SIGNAL_USER_DISCONNECTED", user_id, client_session_id, app)
```

Comment:
shall these signals be fire & forget?

Reply:
the message posting to the websocket is I think